project/civiwiki/settings/project.py

```diff
@@ -0,0 +1,59 @@
+import environ
+
+env = environ.Env()
+
+# reading .env file
+environ.Env.read_env()
+
+# CORS Settings
+CORS_ORIGIN_ALLOW_ALL = True
+
+# API keys
+SUNLIGHT_API_KEY = env.str("SUNLIGHT_API_KEY")
+GOOGLE_API_KEY = env.str("GOOGLE_MAP_API_KEY")
+PROPUBLICA_API_KEY = env.str("PROPUBLICA_API_KEY", optional=True)
+
+
+# Notification API Settings
+NOTIFICATIONS_SOFT_DELETE = True
+NOTIFICATIONS_USE_JSONFIELD = True
+
+
+BASE_DIR = environ.Path(__file__)   # get root of the project
+
+if env('DJANGO_HOST') :
+    DJANGO_HOST = env.Str("DJANGO_HOST")
+else:
+    DJANGO_HOST = 'LOCALHOST'
+
+
+# SSL Setup
+if DJANGO_HOST is not 'LOCALHOST':
+    SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+    SECURE_SSL_REDIRECT = True
+    SESSION_COOKIE_SECURE = True
+    CSRF_COOKIE_SECURE = True
+
+
+
+WSGI_APPLICATION = 'civiwiki.wsgi.application'
+
+
+# Apex Contact for Production Errors
+ADMINS = [('Development Team', 'dev@civiwiki.org')]
+
+
+# AWS S3 Setup
+
+public_root = BASE_DIR.path('public/')
+
+if env('AWS_STORAGE_BUCKET_NAME'):
```

Comment:
These falsy checks are somewhat confusing, as they rely on `env()` returning `None` for missing values.  Try to make it more clear what this is checking for.